.gitmodules

```diff
@@ -1,6 +1,8 @@
 [submodule "third_party/libtock-rs"]
 	path = third_party/libtock-rs
 	url = https://github.com/tock/libtock-rs
+	ignore = dirty
```

Comment:
Why is this necessary now, and wasn't before?

Reply:
Without this `git status` considers `third_party/tock` to be modified. Since that automatically gets modified during build/deployment I considered ignoring it.

E.g., without any manual modifications to `tock` folder.
```
On branch apdu-parser-kk
Your branch is up to date with 'origin/apdu-parser-kk'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
  (commit or discard the untracked or modified content in submodules)
        modified:   .gitmodules
        modified:   third_party/tock (modified content, untracked content)

no changes added to commit (use "git add" and/or "git commit -a")
```

Reply:
Ah got it. The change makes sense to me, I was just wondering if it's somehow related to this PR. Good to have fresh eyes on this.